cob_bringup_sim/scripts/spawn_object_jungheinrich.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/python
```

Comment:
merge already  done: https://github.com/ipa-srd/cob_simulation/pull/1 
With this version we can define "children" objects, that means if I have a glass on a table , I can just define the relative position between the glass and the table, and the script will calculate the relative position between the glass and the world frame.

table1:
  model:  table
  model_type: urdf
  position: [ -14.6, -28.5, 0]  #table link to world
  orientation: [ 0, 0, 1.5708]
  children:
    glass1:
      model:  glass
      model_type: urdf
      position: [ 0, 0, 1]  #glass link to table link
      orientation: [ 0, 0, 0]